packages/documentation/src/content/docs/resources/get-involved.mdx

```diff
@@ -4,27 +4,91 @@ title: Get involved
 
 import { LinkOut } from '@interledger/docs-design-system'
 
-## Participation
+Welcome to the Rafiki community! Whether you're a seasoned developer or just getting started with open source, there are many ways to contribute to the Rafiki project. This guide will help you find your place in our vibrant ecosystem.
 
-- Begin a <LinkOut href="https://github.com/interledger/rafiki/discussions">new discussion or contribute to existing discussions</LinkOut> in the Rafiki GitHub repo.
-- Join the <LinkOut href="https://interledger.slack.com">`interledger.slack.com`</LinkOut> workspace and ask any questions you have in the `#rafiki` channel.
-- Try [running the system locally](/integration/playground/overview) and ask any questions you have in the #rafiki Slack channel.
+## 🚀 Getting started
```

Comment:
Purely a personal decision, but I like "Get started" and "Get help" over "Getting". If you're the opposite, that's fine, we can leave it :)